native/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeConstants.java

```diff
@@ -0,0 +1,59 @@
+    public static final Token SYNTAX_TREE_CLOSE_BRACE = AbstractNodeFactory.createIdentifierToken("}");
+
+    public static final Token SYNTAX_TREE_OPEN_CLOSED_BRACE = AbstractNodeFactory.createIdentifierToken("{|");
+    public static final Token SYNTAX_TREE_CLOSE_CLOSED_BRACE = AbstractNodeFactory.createIdentifierToken("|}");
+    public static final Token SYNTAX_TREE_OPEN_PAREN = AbstractNodeFactory.createIdentifierToken("(");
+    public static final Token SYNTAX_TREE_CLOSE_PAREN = AbstractNodeFactory.createIdentifierToken(")");
+    public static final Token SYNTAX_TREE_OPEN_BRACKET = AbstractNodeFactory.createIdentifierToken("[");
+    public static final Token SYNTAX_TREE_CLOSE_BRACKET = AbstractNodeFactory.createIdentifierToken("]");
+    public static final Token SYNTAX_TREE_EQUAL = AbstractNodeFactory.createIdentifierToken("=");
+    public static final Token SYNTAX_TREE_PIPE = AbstractNodeFactory.createIdentifierToken("|");
+    public static final Token SYNTAX_TREE_SLASH = AbstractNodeFactory.createIdentifierToken("/");
+    public static final Token SYNTAX_TREE_COMMA = AbstractNodeFactory.createIdentifierToken(",");
+
+    public static final Token SYNTAX_TREE_BLANK_LINE = AbstractNodeFactory.createIdentifierToken("\n\n");
+
+    public static final Token SYNTAX_TREE_KEYWORD_IMPORT = AbstractNodeFactory.createIdentifierToken("import ");
+    public static final Token SYNTAX_TREE_KEYWORD_RETURNS = AbstractNodeFactory.createIdentifierToken("returns ");
+    public static final Token SYNTAX_TREE_KEYWORD_RETURN = AbstractNodeFactory.createIdentifierToken("return ");
+    public static final Token SYNTAX_TREE_KEYWORD_STREAM = AbstractNodeFactory.createIdentifierToken("stream ");
+    public static final Token SYNTAX_TREE_IT = AbstractNodeFactory.createIdentifierToken("<");
+    public static final Token SYNTAX_TREE_GT = AbstractNodeFactory.createIdentifierToken(">");
+    public static final TypeDescriptorNode SYNTAX_TREE_VAR_INT =
+            TypeDescriptor.getBuiltinSimpleNameReferenceNode("int");
+    public static final TypeDescriptorNode SYNTAX_TREE_VAR_ANYDATA =
+            TypeDescriptor.getBuiltinSimpleNameReferenceNode("anydata");
+    public static final Token SYNTAX_TREE_QUESTION_MARK = AbstractNodeFactory.createIdentifierToken("?");
+
+    public static final Token SYNTAX_TREE_KEYWORD_RECORD = AbstractNodeFactory.createIdentifierToken("record ");
+    public static final Token SYNTAX_TREE_KEYWORD_IF = AbstractNodeFactory.createIdentifierToken("if ");
+    public static final Token SYNTAX_TREE_KEYWORD_ELSE = AbstractNodeFactory.createIdentifierToken("else ");
+
+    public static final TypeDescriptorNode SYNTAX_TREE_VAR_STRING =
+            TypeDescriptor.getBuiltinSimpleNameReferenceNode("string");
+    public static final TypeDescriptorNode SYNTAX_TREE_VAR_ANY =
+            TypeDescriptor.getBuiltinSimpleNameReferenceNode("any");
```

Comment:
```suggestion
```

Reply:
Seems unused.

Reply:
Updated in [6d8b4a4](https://github.com/ballerina-platform/persist-tools/pull/15/commits/6d8b4a41b9a0ea1002bb906bb2008380cfa7cdb1)